src/desktop/reg-templater.js

```diff
@@ -64,27 +66,46 @@ function getTemplate(
 
 	const r = local ? hkcu : hklm
 	return [
-		{root: sw_clients_mail(r), value: client_template},
-		{root: sw_cls(r), value: {mailto: mailto_template, "tutanota.Mailto": mailto_template}},
-		{root: hkcr, value: {mailto: mailto_template, "tutanota.Mailto": mailto_template}},
-		{root: sw_reg_apps(r), value: {"tutanota": "SOFTWARE\\\\tutao\\\\tutanota\\\\Capabilities"}},
-		{root: sw_wow_reg_apps(r), value: {"tutanota": "SOFTWARE\\\\Wow6432Node\\\\tutao\\\\tutanota\\\\Capabilities"}},
-		{root: sw(r), value: capabilities_template},
-		{root: sw_wow(r), value: capabilities_template}
+		{
+			root: `${r}\\SOFTWARE\\Clients\\Mail`,
+			value: client_template
+		},
+		{
+			root: `${r}\\SOFTWARE\\CLASSES`,
+			value: {mailto: mailto_template, "tutanota.Mailto": mailto_template}
+		},
+		{
+			root: hkcr,
+			value: {mailto: mailto_template, "tutanota.Mailto": mailto_template}
+		},
+		{
+			root: `${r}\\SOFTWARE\\RegisteredApplications`,
+			value: {"tutanota": "SOFTWARE\\\\tutao\\\\tutanota\\\\Capabilities"}
+		},
+		{
+			root: `${r}\\SOFTWARE\\Wow6432Node\\RegisteredApplications`,
+			value: {"tutanota": "SOFTWARE\\\\Wow6432Node\\\\tutao\\\\tutanota\\\\Capabilities"}
+		},
+		{
+			root: `${r}\\SOFTWARE`,
+			value: capabilities_template
+		},
+		{
+			root: `${r}\\SOFTWARE\\Wow6432Node`,
+			value: capabilities_template
+		}
 	]
 }
 
 /**
  * produce a tmp windows registry script to register an executable as a mailto handler
- * @param execPath path to the executable that should be registered
- * @param dllPath path to the mapi dll that handles "Send as Mail..." requests
- * @param logPath path to the directory the mapi dll should put logs in
- * @param tmpPath path to the tmp dir that's managed by tutanota
+ * @param opts {RegistryPaths}
  * @param local set to true if the app was installed per-user
  * @returns {string} registry script
  */
-export function registerKeys(execPath: string, dllPath: string, logPath: string, tmpPath: string, local: boolean): string {
-	const template = getTemplate(esc(execPath), esc(dllPath), esc(logPath), esc(tmpPath), local)
+export function registerKeys(opts: RegistryPaths, local: boolean): string {
```

Comment:
could make "local" part of an object or make enum like `"local" | "global"` because this boolean meaning is not obvious but I think it's also fine as is